heartbeat/galera

```diff
@@ -749,18 +837,21 @@ galera_monitor()
         return $OCF_ERR_GENERIC
     fi
 
-    is_primary
+    check_sync_needed
```

Comment:
We can't poll any mysql property until mysqld finishes its start up, i.e. when pidfile is created.
So we switch the order of check_sync_status is_primary to avoid polling server before SST finishes.